code/modules/mob/holder.dm

```diff
@@ -462,6 +464,13 @@ var/list/holder_mob_icon_cache = list()
 	item_state = "monkey"
 	slot_flags = SLOT_HEAD
 	w_class = ITEMSIZE_NORMAL
+	var/is_normal_monkey = TRUE
+
+/obj/item/holder/monkey/set_contained(var/mob/living/carbon/human/M)
+	..()
+	M.dir = SOUTH //monkeys look better head-on | source: it was revealed to me in a mirror
+	if(is_normal_monkey && istype(M.w_uniform, /obj/item/clothing/under/punpun))
+		item_state = "monkey_fancy"
```

Comment:
This is horribly hardcoded. The better thing to do would be to make a system that decides item_state based on the item the monkey is wearing and only changes the state if it exists.